pipeline/git_test.go

```diff
@@ -156,3 +156,60 @@ func TestUpdateAllPipelinesHundredPipelines(t *testing.T) {
 		t.Fatal("log output did not contain error message that the repo is up-to-date.: ", b.String())
 	}
 }
+
+func TestGetAuthInfo(t *testing.T) {
```

Comment:
Would you be so kind as to extract these into three smaller tests? Or, if you want the setup, than use `t.Run("bla", func() {})` in order to have three legs in one test while still using the setups.